streaming/base/format/base/writer.py

```diff
@@ -378,6 +357,7 @@ class SplitWriter(Writer):
             Defaults to ``None``.
         size_limit (int, optional): Optional shard size limit, after which point to start a new
             shard. If None, puts everything in one shard. Defaults to ``1 << 26``.
+        **kwargs (Any): Additional settings for the Writer
```

Comment:
trailing .

Reply:
Addressed as part of https://github.com/mosaicml/streaming/pull/171/commits/5b5c10450637d4ccdbc8b5c2403a7d18f89b169b.